app/services/support_interface/ministerial_report_candidates_export.rb

```diff
@@ -15,26 +15,43 @@ def call(*)
 
       export_rows[:split] = column_names
 
-      application_forms.find_each do |application|
-        next if application.phase == 'apply_2'
+      subject_report = {}
 
+      application_forms.find_each do |application|
         subjects = determine_subjects(application)
 
-        states = if candidate_has_a_successful_apply_2_application?(application)
+        states = if candidate_has_a_viable_apply_2_application?(application)
```

Comment:
@duncanjbrown This seems to be the source of some discrepancies. It makes sense to me to pick the latest viable application. This seems to be what is happening in the case of at least one candidate that has 3 applications (and 5 application choices). The latest application choice was `withdrawn` but all the others were `rejected`. TAD seem to be putting this in the `rejected` column whereas we are putting it under `withdrawn`. Isn't that right?

Reply:
I think that is right